cli/dcoscli/data/help/package.txt

```diff
@@ -56,6 +57,8 @@ Options:
         Command line only.
     --config
         Print the configurable properties of the `marathon.json` file.
+    --dry-run, -n
```

Comment:
I'm not sure of the value of the dry-run here. If your import fails because of an error you can always add again without any consequences